src/shared/route/useRouteNavigation.ts

```diff
@@ -68,7 +68,7 @@ export const useRouteNavigation = () => {
       body,
     }: {
       companyId: string;
-      body?: PostRouteQuery;
+      body?: PostRouteQuery | null;
```

Comment:
null을 허용하면 아래와 같이 바꿔도 되지 않을까요?
```suggestion
      body: PostRouteQuery | null;
```

그리고 위쪽에서 body로 처리되지 않는 이유가 state에서 body를 스프레드 문법으로 넣고 있어서 null값이 안 나올수도 있을 거 같은데 혹시 빈 body를 전달하면 값이 어떻게 전달되나요?